doc/source/release.rst

```diff
@@ -2,6 +2,8 @@
 Release Notes
 *************
 
+.. include:: ../release/1.14.4-notes.rst
+.. include:: ../release/1.14.3-notes.rst
```

Comment:
Ah, I missed this in 1.14.3. Shall I add a paragraph to the release walkthrough about which release/contributor files should be updated?

Reply:
Sure. I'm too close to the process to see those things and anything that helps the next person to make releases is good.